src/main.rs

```diff
@@ -78,7 +78,7 @@ fn create_app<'a, 'b>(
     api_server_path: &'a str,
 ) -> App<'a, 'b> {
     App::new("cloud-hypervisor")
-        .version(crate_version!())
+        .version(concat!(crate_version!(), env!("BUILT_VERSION")))
```

Comment:
How do we expect the `BUILT_VERSION` environment variable to be set?
Is `println!("cargo:rustc-env=BUILT_VERSION= commit: {}", built_version);` responsible for doing so because this is the output from the `main()` function in `build.rs`?
If that's the case, I think this `println!()` would deserve a comment to explain what it really does!

Reply:
@sboeuf see: https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env

Reply:
And: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

I agree we could comment from `main.rs` to explain where this variable is coming from.

Reply:
Yes I did look at it, but I still think we should leave a comment in the `build.rs` file as this is a pretty special behavior.
Other than that, the patch looks great :)

Reply:
I would also add a comment in `main.rs`, referring back to build.rs.